api/src/routes/user.ts

```diff
@@ -6,18 +6,22 @@ const router = Router();
 router.get('/', (req: Request, res: Response, next: NextFunction) => {
 	User.findAll()
 		.then((users) => {
-			res.send(users);
+			return res.send(users);
 		})
-		.catch((error) => next(error));
+		.catch((error) => {
+			return res.status(404).send(error);
```

Comment:
ERROR 500

Reply:
404?
https://c.tenor.com/k1wbOgEPazIAAAAS/the-rock-sus-meme-the-rock-sus.gif